lisa/analysis/latency.py

```diff
@@ -175,7 +175,7 @@ def df_runtimes(self, task):
 
     @df_latency_wakeup.used_events
     def plot_latencies(self, task, wakeup=True, preempt=True, threshold_ms=1,
-                       filepath=None, axis=None):
+                                                                    **kwargs):
```

Comment:
That's a lot of whitespace...

Reply:
That's to align with the previous line (tabs are for indentation, spaces for alignment). I don't mind aligning with the opening paren instead of the closing one if we prefer that.